classes/question/bank/studentquiz_bank_view.php

```diff
@@ -676,9 +677,9 @@ protected function parse_subsort($sort): array {
     /**
      *  Return the all the required column for the view.
      *
-     * @return \question_bank_column_base[]
+     * @return array \question_bank_column_base[]
```

Comment:
Acutally, 

`@return \question_bank_column_base[]`

is valid PHPdoc, and gives more information than just array, so better to use that.